src/3p-frame.js

```diff
@@ -196,8 +196,8 @@ export function getBootstrapBaseUrl(parentWindow, opt_strictForUnitTest) {
  * @return {string}
  */
 function getDefaultBootstrapBaseUrl(parentWindow) {
-  if (getMode().localDev || parentWindow.AMP_TEST) {
-    const prefix = parentWindow.AMP_TEST ? '/base' : '';
+  if (getMode().localDev || getMode().test) {
```

Comment:
@cramforce yes, will continue working on that. will need to expose it at module level instead of a value from `getMode()`. was struggling with getting tests to pass with my DCE changes but should be close.